### ISSUE TITLE
Cant redirect to profile page after submiting

### ISSUE THREAD

**Issue by raakshash** [RisingStack/nodehero-authentication#3]
It is not redirecting to profile page after submit

**Comment by SiarheiRylach** [comment]
I have resolved this issue. You need to delete lines 44-47 in authentication/init.js:
 if (password !== user.password) {
    return done(null, false);
}

**Comment by mitrofanzzz** [comment]
Nope, isn't work...

**Comment by melikhov-dev** [comment]
This is because you use the credentials from the example, but the correct credentials:
```
    username: 'test-user',
    password: 'my-password'
```

**Comment by mitrofanzzz** [comment]
Nope, i am not so silly, i found user n pass from source... But it's not redirecting me... How i can debug it?

**Comment by melikhov-dev** [comment]
You can run the application with the `--inspect` flag and debug it with Chrome DevTools. In my case, everything works correctly.

**Comment by mitrofanzzz** [comment]
Checked twice, on different PC (Debian, Ubuntu) isn't work for me.
